app/build.gradle

```diff
@@ -44,11 +44,11 @@ static def getProperty(@Nullable Properties properties, String name) {
 }
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion '27.0.3'
+    compileSdkVersion 28
+    buildToolsVersion '28.0.0'
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 27
+        targetSdkVersion 28
 
         renderscriptTargetApi 27
```

Comment:
Might have to bump this too.

Reply:
Sure! I didn't want to commit just in case it may cause any issues but it's been fine on my local testing so far (all on P though 🤔)